package.json

```diff
@@ -51,7 +51,9 @@
     "semver": "5.1.0",
     "shelljs": "0.6.0",
     "webpack": "1.12.14",
+    "webpack-dev-middleware": "1.5.1",
```

Comment:
Should we start dropping the patch versions in this file? I think if this is set to `1.5` then greenkeeper will automatically update it without a PR until it hits `1.6.x`. Maybe the PRs are better but it seems like it would be convenient to have most stuff land automatically.